Classes/Event/EventWithMetadata.php

```diff
@@ -1,59 +0,0 @@
-<?php
-namespace Neos\Cqrs\Event;
-
-/*
- * This file is part of the Neos.Cqrs package.
- *
- * (c) Contributors of the Neos Project - www.neos.io
- *
- * This package is Open Source Software. For the full copyright and license
- * information, please view the LICENSE file which was distributed with this
- * source code.
- */
-
-use TYPO3\Flow\Annotations as Flow;
-
-/**
- * Wrapper container for a domain event that is enriched with metadata.
- * This class is immutable, if the event contained is implemented immutable.
- *
- * @Flow\Proxy(false)
- */
-final class EventWithMetadata
```

Comment:
This class was used to transport events & metadata to the EventPublisher and from the Event Store.
For the former we don't need the metadata any longer (see above) and the latter has been replaced with `EventTransport` that contains the converted event & it's raw representation from the store including version and type